### ISSUE TITLE
Make item cap limit configurable + UI notification

### ISSUE THREAD

**Issue by mrosseel** [pkamenarsky/atea#25]
The last line of this file is not shown in the menu, this is just a minimal example showing this behaviour, also had it in the middel of the file with other entries.

Contents of the file:
[aeg] idcard  oplossen, zie pdf in mapje aeg\ => email antwoorden verzamelen
[aeg] id probleem: mijn id toevoegen aan doc
[aeg] integreer SN en SP in ods sheet
betaal 16 euro aan jo 
betaal 20 euro aan kurt
betaal 5 euro aan joyce
[ra] rekening bart sturen ouders ==> heb ik die al terugbetaald ???
[appt] compromis 3x afprinten
[dandi] mail about dandi
    Hallo iedereen,
[eon] aankoopprijs auto!
[ra] mailen ramen

**Comment by pkamenarsky** [comment]
Yes, this is by design - Atea only takes the first 10 items per project / priority so as to keep the menu clean. You should probably prioritize items (use empty lines) and reorder them accordingly - you can't concentrate on 50 items at once anyway.

Still, I should probably make that configurable.

**Comment by mrosseel** [comment]
Hi, no issue with the design principle, although some kind of warning might have prevented me from filing a bug ;).
Maybe add 'Priority 1 - bla - WARNING: too many tasks' or something more poetic, this might make the restriction more intuitive. 

Adding extra config on top of that would indeed be a nice bonus.

**Comment by pkamenarsky** [comment]
No issue with the bug report either - we're all brainstorming ideas for the next version, and this is definitely something that can be improved upon :)

**Comment by maebert** [comment]
Idea: have an input field at the top that lets you filter all tasks and projects, much like spotlight.

If you automatically give it the focus when open and use a system-wide hotkey (#10), you'll instantly win all us keyboard-only geeks :)

**Comment by pkamenarsky** [comment]
Good idea, although it is already possible to filter tasks by just typing since this is a default feature of MacOS. The system wide hotkey would take a bit of work though, don't know when I'll have time to hack on Atea though :/

edit: not filter but rather select tasks

**Comment by maebert** [comment]
True. By filter I mean having visual feedback, ie. the list of tasks and projects shortens as you type as everything that doesn't fuzzy-match your input gets hidden. That comes in handy if you have several similarly named tasks (e.g. "Fix bugs") for different projects, so typing `ateafx` would get you to `[Atea] Fix bugs`, but not to `[Universe] Fix Bugs`.

**Comment by pkamenarsky** [comment]
Absolutely agree, my comment was just meant as a "quick-fix" for right now. Still haven't found a tool that would allow a system wide hotkey for a status bar icon though.

Apart from that, both advertising a service (for the hotkey) and a text field would require native JNI hacking.
